assets/stylesheets/_components.scss

```diff
@@ -177,6 +172,11 @@
 @import 'my-sites/draft/style';
 @import 'my-sites/drafts/style';
 @import 'my-sites/exporter/style';
+@import 'my-sites/invites/invite-form-header/style';
+@import 'my-sites/invites/invite-header/style';
+@import 'my-sites/invites/invite-accept/logged-in/style';
```

Comment:
So, to clarify, we either need to bring the `logged-in` and `logged-out` into the `my-sites/invites` directory, or do the sub-components?